group.yml

```diff
@@ -177,3 +177,22 @@ image_build_log_scanner:
   - ppc64le.log
   - task_failed.log
   - orchestrator.log
+
+# Images which are NOT to be synced
+blacklist:
```

Comment:
I could argue whether `blacklist` or even `sync_blacklist` is specific enough. `blacklist` doesn't say what's being blacklisted. A `sync` sounds a lot like a `push` and we still want to `images:push`.

Something based on the function these images have/don't would help. Maybe `blacklist_image_release`?

Reply:
Agree with @sosiouxme here. It's a good point. Not that I'm not guilty of it, but unless this is truly more general I agree it should be more specifically named.